### PR TITLE
Connect add-book form to store and add Author and Book data to database

### PR DESCRIPTION
**Pull request by mig824**
Gets the user's inputs from the add-book form and sends it to the redux store, it also sends a request to the google books api and adds the relevant data from that into the store as well

### CODE REVIEW

**Comment by mig824** [comment]
_It's kind of a big PR , sorryyyy_

Notable changes:
- Results from Google API are displayed
- Author and Book can be added to database
- CSS stuff for `/add-book` page
- Removed `tag` logic for now

File: src/types/BookRequest.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface BookRequest {
+  userId: string;
+  googleId: string;
+  title: string;
```
Comment (matt-ramotar) [inline src/types/BookRequest.ts:4]: This should have a description field

File: src/lib/prepareBookForDB.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  } catch (error) {
+    console.log(error);
+    throw error;
+  }
+
+  return {
+    userId: user.id,
+    googleId: book.id,
+    title: book.volumeInfo.title,
+    coverImage: book.volumeInfo.imageLinks.thumbnail,
```
Comment (matt-ramotar) [inline src/lib/prepareBookForDB.ts:27]: This should also capture description

**Review by matt-ramotar — CHANGES_REQUESTED** [review]
LG! Minor changes to include book description

**Review by matt-ramotar — APPROVED** [review]

**Pull request merged**